setup.py

```diff
@@ -27,4 +37,4 @@
       tests_require=[
           'pytest'
       ]
-     )
+      )
```

Comment:
It made flake8 stop complaining (it wasn't failing, but it removed a warning). We can change back _shrug_ I like the previous way it was better personally